examples/iris.jl

```diff
@@ -7,22 +7,22 @@ using StatsBase
 iris = dataset("datasets", "iris")
 
 # observations in columns and variables in rows
-X = array(iris[:, 1:4])'
+X = convert(Array, iris[:, 1:4])'
 p, n = size(X)
 # by default species is a PooledDataArray,
 y = [species for species in iris[:, 5]]
 
 # how much data use for training
 train_frac = 0.9
-k = int(floor(train_frac * n))
+k = round(Int, floor(train_frac * n))
 idxs = randperm(n)
 train = idxs[1:k]
 test = idxs[k+1:end]
 
 model = GaussianNB(unique(y), p)
-fit(model, X[:, train], y[train])
+NaiveBayes.fit(model, X[:, train], y[train])
```

Comment:
Hmm, I see you qualified `NaiveBayes.fit` to avoid conflicts with `RDatasets.fit`, but in fact, they both are supposed to extend `StatsBase.fit`. Either I missed it initially or these changes have been lost in some refactoring session, so sorry for the confusion. 

The proper fix is to add `import StatsBase: fit, predict` to `NaiveBayes.jl`. Could you please remove this specific change from the commit so that I could fix it?